opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml

```diff
@@ -185,7 +147,7 @@ paths:
       security:
         - sessionCookie: []
 
-  /v1/consent/{auth-id}/fromAspsp/{redirectState}/ok/{fromAspspRedirectCode}:
+  /v1/consent/{auth-id}/fromAspsp/ok/{redirectState}:
```

Comment:
Suggest to rephrase endpoint to:
`/v1/consent/{auth-id}/fromAspsp/ok/{redirectState}` -> `/v1/consent/{auth-id}/fromAspsp/{redirectState}/ok

Reply:
endpoint rephrased